### ISSUE TITLE
Word list is quite short

### ISSUE THREAD

**Issue by FSMaxB** [macaullyjames/correcthorsebatterystaple#3]
Your wordlist contains only 2048 words, which isn't that long. It will hold up to brute force attacks for quite some time if you only allow 1000 guesses per second, but if the attacker has access to the hashes and a fast hash function has been used, it won't be that good anymore.

You could for example create a wordlist here: http://app.aspell.net/create

**Comment by macaullyjames** [comment]
You're right of course, but I'm not convinced that we should worry too much about offline cracking. As long as people don't reuse their passwords the only thing an attacker gains by cracking the hash is access to the service that it was used on... But then again, if an attacker has access to the hash it's pretty safe to say that the entire service is compromised anyway.

There might be some edge case that I'm neglecting here though 🤔

**Comment by FSMaxB** [comment]
I guess if people feel that their password is secure, they are going to use it on even more sites.

Also these kinds of passwords are harder to remember than the passwords regular people would normally use, so they have even more incentive to reuse it.

In security we should protect people from themselves instead of blaming them (because blaming them has never worked and never will).

**Comment by macaullyjames** [comment]
I suppose.

This could be solved with a bit of clever UX. Instead of generating generic passwords we could generate "app specific passwords".  They'd be the same normal, randomly generated passwords as now, but by forcing the user to choose the service they want to use the password for we're reinforcing the idea that the password is for that specific service _only_. It'd be natural to create one password for dropbox, one for email etc :)

**Comment by FSMaxB** [comment]
Interesting idea. I would have to think about that.
